packages/alpha/vite.config.js

```diff
@@ -2,11 +2,14 @@ import { resolve } from 'node:path';
 import { defineConfig } from 'vite';
 import dts from 'vite-plugin-dts';
 
+const externals = ['@cognite/sdk', '@cognite/sdk-core'];
+
 export default defineConfig({
   plugins: [
     dts({
       exclude: ['**/__tests__/**/*'],
       entryRoot: '.',
+      aliasesExclude: externals,
```

Comment:
Without this, then the typescript declaration files will use relative paths instead of `@cognite/sdk-core`

Reply:
Is this because they are defined as paths in tsconfig?

Reply:
Maybe